index.bs

```diff
@@ -318,6 +318,23 @@ but in some cases the feature should also be detectable
 in the language where it is used
 (such as ''@supports'' in CSS).
 
+<h3 id="restriction-options">Use appropriate methods for restricting features</h3>
+
+The Web has a number of somewhat-related features:
```

Comment:
should we add "for example:" (see [below](https://github.com/w3ctag/design-principles/pull/363/files#r1171993466) )